### ISSUE TITLE
Facebook Marketing API failing asking for upgrade to v15

### ISSUE THREAD

**Issue by louismorinkar** [airbytehq/airbyte#18167]
## Environment
- **Airbyte version**:  0.40.3
- **OS Version / Instance**: GCP n2
- **Deployment**: Docker
- **Source Connector and version**: Facebook Marketing 0.2.69
- **Destination Connector and version**: 
- **Step where error happened**: Setup new connection 

## Current Behavior
The connection tests failed, with an error message "FacebookAPIException: You are calling a deprecated version of the Ads API. Please update to the latest version: v15.0.')"

## Expected Behavior
The connection tests should succeed

## Logs

[server-logs.txt](https://github.com/airbytehq/airbyte/files/9821968/server-logs.txt)
[scheduler-logs.txt](https://github.com/airbytehq/airbyte/files/9821969/scheduler-logs.txt)

## Steps to Reproduce
1. Create a connector with Facebook Marketing
2. Test the connection

## Are you willing to submit a PR?
No

**Comment by YowanR** [comment]
@YuliiaNahoha Can you please take a look at this ASAP? Thanks! 🙏

**Comment by grubberr** [comment]
@louismorinkar I don't understand where I can see this error message
`FacebookAPIException: You are calling a deprecated version of the Ads API. Please update to the latest version: v15.0.`
I downloaded both logs: `server-logs.txt`, `scheduler-logs.txt`
I did not see such error message in both log files

**Comment by louismorinkar** [comment]
Hi, the message is shown in the UI : 
![The connection tests failed](https://user-images.githubusercontent.com/114916425/196747180-9fd11fcd-1ee4-4368-a3db-8f4e58a4979a.png)

**Comment by grubberr** [comment]
@louismorinkar can you please put here a  log of failed "check" command 
You need to press [Set up source] and get logs from UI

**Comment by louismorinkar** [comment]
I don't understand why the logs can't be found in the scheduler / server logs but here are the logs :

```
7:07PM 05/22
2022-10-20 07:27:25 INFO i.a.w.t.TemporalAttemptExecution(get):105 - Docker volume job log path: /tmp/workspace/90eac019-f747-4435-bf63-4b065557b452/0/logs.log
2022-10-20 07:27:25 INFO i.a.w.t.TemporalAttemptExecution(get):110 - Executing worker wrapper. Airbyte version: 0.40.3
2022-10-20 07:27:25 INFO i.a.c.f.EnvVariableFeatureFlags(getEnvOrDefault):49 - Using default value for environment variable USE_STREAM_CAPABLE_STATE: 'false'
2022-10-20 07:27:25 INFO i.a.c.i.LineGobbler(voidCall):83 - Checking if airbyte/source-facebook-marketing:0.2.69 exists...
2022-10-20 07:27:25 INFO i.a.w.p.DockerProcessFactory(create):108 - Creating docker job ID: 90eac019-f747-4435-bf63-4b065557b452
2022-10-20 07:27:25 INFO i.a.w.p.DockerProcessFactory(create):163 - Preparing command: docker run --rm --init -i -w /data/90eac019-f747-4435-bf63-4b065557b452/0 --log-driver none --name source-facebook-marketing-check-90eac019-f747-4435-bf63-4b065557b452-0-ootie --network host -v airbyte_workspace:/data -v /tmp/airbyte_local:/local -e DEPLOYMENT_MODE=OSS -e USE_STREAM_CAPABLE_STATE=false -e AIRBYTE_ROLE= -e WORKER_ENVIRONMENT=DOCKER -e WORKER_JOB_ATTEMPT=0 -e WORKER_CONNECTOR_IMAGE=airbyte/source-facebook-marketing:0.2.69 -e AIRBYTE_VERSION=0.40.3 -e WORKER_JOB_ID=90eac019-f747-4435-bf63-4b065557b452 airbyte/source-facebook-marketing:0.2.69 check --config source_config.json
2022-10-20 07:27:25 INFO i.a.c.i.LineGobbler(voidCall):83 - airbyte/source-facebook-marketing:0.2.69 was found locally.
2022-10-20 07:27:27 ERROR i.a.w.i.DefaultAirbyteStreamFactory(internalLog):95 - Giving up call(...) after 1 tries (facebook_business.exceptions.FacebookRequestError: 
  Message: Call was not successful
  Method:  GET
  Path:    https://graph.facebook.com/v14.0/act_1379063265742873/
  Params:  {}
  Status:  400
  Response:
    {
      "error": {
        "message": "(#2635) You are calling a deprecated version of the Ads API. Please update to the latest version: v15.0.",
        "type": "OAuthException",
        "code": 2635,
        ```

**Comment by grubberr** [comment]
@louismorinkar

I suspect that problem can be in Facebook application settings
Can you look on your facebook application settings like:


https://developers.facebook.com/ -> Settings -> Advanced -> Upgrade API version

maybe we will find something interesting here ?

**Comment by grubberr** [comment]
@louismorinkar @YowanR 
it looks like you someway restricted your facebook application to use only API v15.
One option to re-create facebook application with lower API version.
I am not sure we will migrate to API v15 pretty fast.

**Comment by yoanyahemdi** [comment]
Hello @grubberr 
It is not possible to downgrade to lower version on Facebook App, even though by creating a new app from scratch.
It only offers v15.

**Comment by BrianGiraldo** [comment]
Facebook only enables applications that have consumed older versions of the API to use them, as yoanyahemdi indicates, new applications are limited to using the latest version of the published API.

It would be great to be able to choose the API version in the Airbyte interface when configuring the feed.

**Comment by grubberr** [comment]
@YowanR 
it looks like this is problem only for new open-source customers.
Probably we need to schedule some time to rewrite this connector on Facebook API v15 in future.

**Comment by girarda** [comment]
@YowanR FYI: the SDK for Facebook API v15 isn't available yet. Latest one is still v14.0 https://pypi.org/project/facebook-business/

**Comment by BrianGiraldo** [comment]
> @YowanR FYI: the SDK for Facebook API v15 isn't available yet. Latest one is still v14.0 https://pypi.org/project/facebook-business/

v15.0.0 is available now

**Comment by girarda** [comment]
Closing because it's been fixed in 0.2.70.

**Comment by louismorinkar** [comment]
Thanks !
